docs/public-share.md

```diff
@@ -0,0 +1,54 @@
+# Public share
+
+## Compendium
+
+__Stability:__ 0 - subject to changes
```

Comment:
please remove this statement, we're not providing that information at this level anymore because it will be outdated anyway